ptoss-2-funcao-joaquim-isPasswordValid.md

```
@@ -0,0 +1,213 @@
+corpo real nunca é executado por nenhum teste, e a cobertura real é **0%**.
+
+Isso torna esta função ideal para o trabalho: a cobertura "antes" é 0% e a
+"após" será 100%, gerando um delta claro.
+
+## 3. Projeto dos Casos de Teste (caixa-branca-primeiro)
+
+### 3.1. Tabela de decisões e condições
+
+| ID | Decisão (linha) | Condições | True | False |
+| --- | --- | --- | --- | --- |
+| D1 | `password.length >= 7 && (!strict || password.length > 14)` (L13) | A=`len>=7`, B=`!strict`, C=`len>14` | `min = true` | `min = false` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)